docs/beyond_basics/101-164-dataladdening.rst

```diff
@@ -0,0 +1,141 @@
+
+Prior to transforming your project, **we recommend to create a copy of it**.
+We don't believe there is much that can go wrong from the software-side of things, but data is precious, so if its your first time using DataLad, better be safe than sorry.
+
+Step 1: Planning
+^^^^^^^^^^^^^^^^
+
+The first step to DataLad-ify your project is to turn it into one or several nested datasets.
+Whether you turn a project into a single dataset or several is dependent on the current size of your project and how much you expect it to grow overtime, but also on its contents.
+You can find guidance on this in paragraph below.
+
+The next step is to save dataset contents.
+You should take your time and invest thought into this, as this determines the looks and feels of your dataset, in particular the decision on which contents should be saved into :term:`Git` or :term:`git-annex`.
+The section :ref:`symlink` should give you some necessary background information, and the chapter :ref:`chapter_config` the relevant skills to configure your dataset appropriately.
+You should consider the size, file type and modification frequency of files in your decisions as well as potential plans to share a dataset with a particular third party infrastructure.
+
+Step 2: Dataset creation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Transforming a directory into a dataset is done with :command:`datalad create --force`.
+The ``-f``/``--force`` option enforces dataset creation in non-empty directories.
+Consider :ref:`applying procedures <procedures>` with ``-c <procedure-name>`` do apply configurations that suit your use case.
+
+.. findoutmore:: What is my directory is already a Git repository?
+
+   If you want to transform a Git repository to a DataLad dataset, a :command:`datalad create -f` is the way to go, too, and completely safe.
+   Your Git history will stay intact and will not be tempered with.
+
+If you want to transform a series of nested directories into nested datasets, continue with :command:`datalad create -f` commands in all further subdirectories.
+
+.. findoutmore:: One or many datasets?
+
+   In deciding how many datasets you need, try to follow the benchmarks in chapter :ref:`chapter_gobig` and the yoda principles in section :ref:`yoda`.
+   Two simple questions can help you make a decision:
+
+   #. Do you have independently reusable components in your directory, for example data from several studies, or data and code/results? If yes, make each individual component a dataset.
+   #. How large is each individual component? If it exceeds 100-200k files, split it up into smaller datasets. The decision on where to place subdataset boundaries can be guided by the existing directory structure or by common access patterns, for example based on data type (raw, processed, ...) or subject association. One straightforward organization may be a top-level superdataset and subject-specific subdatasets, mimicking the structure chosen in the use case :ref:`usecase_HCP_dataset`.
```

Comment:
I'd say ~100k files. 500k are also fine under some circumstances, but 100k is pretty heavy already in the general case.